app/models/option_node.rb

```diff
@@ -240,8 +240,12 @@ def arrange_as_json(hash = nil)
         # this fix, but they should be much rarer because folks likely won't be trying to delete an option
         # that has so many answers.
         #
-        # This should ideally consult the `destroy` permission for option_node in the authorization system,
-        # but it all needs to be refactored anyway; it's not worth doing more gross things just for this.
+        # This should ideally consult the `destroy` permission for option_node in the authorization system.
+        # But Ability is not easily accessed from models. Also, the option set form is due for a refactor
+        # and will likely not need JSON serialization in the future. So for now we'll just duplicate
+        # the logic to test if removal is allowed. See the `user_independent_permissions` method
+        # in Ability for the source of this.
+        # branch[:removable] = !data? && !in_use?
         branch[:removable] = true
```

Comment:
Again drawing attention to this area with rebase conflicts:

`= true` was on develop. I left your new code commented out here, otherwise it would not exist in git history.

Reply:
Yes I think it's fine.

Reply:
I removed the commented code. `= true` is the most recent.